integrations/gitlab.js

```diff
@@ -8,49 +8,47 @@ const { gitIssues } = require('../utils/api/gitlab')
  */
 async function listIssues(authData) {
   const { gitlabToken, projectId } = authData
-  sdk.log('')
-  ux.spinner.start('Retrieving GitLab issues')
+  await ux.print('')
+  await ux.spinner.start('Retrieving GitLab issues')
   const issues = await gitIssues(gitlabToken, projectId)
-  ux.spinner.stop('Done!')
+  await ux.spinner.stop('Done!')
 
   if (!issues.length) {
-    sdk.log(ux.colors.magenta('\nNo issues found!'))
+    await ux.print(ux.colors.magenta('\nNo issues found!'))
     return
   }
 
-  sdk.log(ux.colors.magenta('\nWe found the following open issues:\n'))
-  issues.map(printIssue)
+  await ux.print(ux.colors.magenta('\nWe found the following open issues:'))
```

Comment:
I think you can combine these lines into one print (it'll show up better on Slack that way) something like:
```
const { magenta } = ux.colors
const titleStr = magenta('\nWe found the following open issues:')
const issuesStr = issues.map(printIssue)
await ux.print(`${titleStr}\n${issuesStr.join('')}`)
```